autorest/date/unixtime_test.go

```diff
@@ -1,3 +1,5 @@
+// +build go1.7
```

Comment:
I'm a little confused on this one, did I use a feature that is only available in 1.7+ while implementing this? Should I rewrite portions so that Unixtime is available to 1.6 users as well? If not, does that mean packages that require Unixtime would also need the 1.7 build tag?